cmd/slinky/main.go

```diff
@@ -77,6 +80,20 @@ func init() {
 		"6060",
 		"Port for the pprof server to listen on.",
 	)
+	rootCmd.Flags().StringVarP(
+		&logLevel,
+		"log-level",
+		"",
+		"info",
+		"Log level (debug, info, warn, error, dpanic, panic, fatal).",
+	)
+	rootCmd.Flags().StringVarP(
```

Comment:
Why do we need this? @Eric-Warehime is this standard practice? My intuition is that all infra deployments assume logs are piped to stdout, and will redirect wherever they want from there (i.e in solve)

Reply:
see [here](https://github.com/skip-mev/solve/blob/main/logging/logging.go#L21)

Reply:
more of a utility thing than anything else